projects/m2/019-morse-code/js/index.js

```diff
@@ -0,0 +1,77 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function convertToMorseCode(message, dictionary){
+    message = message.toUpperCase();
+    message = message.replaceAll(' ', '');
+
+    let convertedMessage = '';
+
+    for (let i = 0; i < message.length; i++) {
+        const letter = message[i];
+
+        if(Object.keys(dictionary).includes(letter)){

```

Comment:
You could also use the in-operator in these cases ;-)

Reply:
Hi, I fixed the exercise.